fonts.css

```diff
@@ -0,0 +1,12 @@
+@font-face {
+  font-family: "Gotham Pro";
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5699758)